onadata/libs/serializers/dataview_serializer.py

```diff
@@ -228,7 +229,7 @@ def get_last_submission_time(self, obj):
 
         return None
 
-    def get_instances_with_geopoints(self, obj):
+    def get_instances_with_geopoints(self, obj) -> bool:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5558448)